src/store/index.js

```diff
@@ -5,6 +5,25 @@ Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
+    materials: {
+      wood: 0,
+      clay: 0,
+      iron: 0,
+      villagers: 0,
+    },
+    units: {
+      pikes: 0,
+      swords: 0,
+      axes: 0,
+      archers: 0,
+      trebuchets: 0,
+      horses: 0
+    },
+    village: {
+      x: 0,
+      y: 0,
+      points: 0
+    }
```

Comment:
It is good practice to use modules with store. During the development of the application, many getters, states, mutations and actions will come and your file will have 999999999 lines - it will be hard to see what's going on. You should divide this one for example to` materialsModule`, `unitsModule`, `villageModule`. I enclose a link to the vuex documentation
(https://vuex.vuejs.org/guide/modules.html#module-local-state)